src/widgets/hero/ui/hero.tsx

```diff
@@ -6,8 +8,12 @@ export const Hero = () => {
       <div className="main content">
         <div className="title-container">
           <div className="subtitle-container">
-            <h2 className="subtitle">an international community of developers</h2>
-            <h2 className="subtitle">since 2013</h2>
+            <Subtitle fontSize="extraSmall" color="black">
```

Comment:
Personally, I'd prefer the kebab case for values `fontSize="extra-small"`, but let's discuss with the team

Reply:
I agree, missed it, thank you